src/app/service/recap/ListItem.jsx

```diff
@@ -2,16 +2,29 @@ import React from 'react';
 import { Link } from 'react-router-dom';
 
 import Icon from '../../../components/icon';
+import { SERVICE_FIELDS } from '../../serviceForm/routes';
+import { DOCUMENT_FIELDS } from '../../documents/routes';
 
 import './ListItem.css';
 
+function filterRoutes(service, route){
+  return service.metadata[route.metaDataSection].find( metadata => metadata.field_name === route.fieldName)
```

Comment:
I ran eslint, and there are lots of errors, not specifically related to this PR. It seems we have some cleanup to do.

It’s interesting because “npm run dev «  actually runs lint on build, but it seems not to detect our .eslint configuration. Fixing lint in our build would be ideal, because then we do not need to rely on setting up individual IDEs, which may vary. Alternatively, we could set up a Travis CI with this project, which can run lint automatically when new commits are pushed to github. I will create a new issue for this.

Thanks.